tools/static-assets/skel-full/package.json

```diff
@@ -5,6 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
-    "meteor-node-stubs": "~0.2.0"
+    "babel-runtime": "^6.18.0",
```

Comment:
Yeah I thought the same, though if it was fairly trivial to implement an automated method I figure it would be worthwhile